.projenrc.js

```diff
@@ -63,6 +63,7 @@ const project = new cdk.JsiiProject({
   publishToGo: {
     gitUserName: 'cdk8s-automation',
     gitUserEmail: 'cdk8s-team@amazon.com',
+    gitBranch: '1.x',
```

Comment:
Kind of feels like this should be the default in projen if we release from multiple branches, no?